app/controllers/users_controller.rb

```diff
@@ -574,18 +574,21 @@ def my_vote_summary
   end
 
   def vote_summary
-    @votes = Vote.where(recv_user: @user) \
-                 .includes(:post).group(:date_of, :post_id, :vote_type)
-    @votes = @votes.select(:post_id, :vote_type) \
-                   .select('count(*) as vote_count') \
-                   .select('date(created_at) as date_of')
+    @votes = Vote.where(recv_user: @user)
+                 .joins(:post)
```

Comment:
As far as I can see these two diffs are identical (excepting blank space) except this line - changed from an `includes` to a `joins`. Unless Rails has changed something since I last knew about it, `joins` _doesn't_ preload the referenced table, which is what `includes` does. What's the reasoning behind this change?

Reply:
Fixed as per discussion - I feared that'll require a rewrite of the query, but after looking into it, I realized that the changed version does not require `joins` and can make do with `includes` without sacrificing the fix.